theme/lumo/vaadin-context-menu.html

```diff
@@ -5,6 +5,24 @@
 <dom-module id="lumo-context-menu-overlay" theme-for="vaadin-context-menu-overlay">
   <template>
     <style include="lumo-menu-overlay">
+      [part="content"] {
+        background-color: #fff;
```

Comment:
This should be removed. `lumo-menu-overlay` specifies the necessary background colors.

Reply:
Done.